git-hub

```diff
@@ -318,10 +411,18 @@ def json_to_toml(username, repo, pr, token, open_or_closed):
     response = urlopen(request)
     response = json.loads(response.read().decode('utf-8'))
     to_write = to_write + "['{0} pull requests'.{1}]\n".format(open_or_closed, pr.number)
-    to_write = to_write + 'user = "{0}" \n'.format(pr.user.name)
+    to_write = to_write + 'title = "{0}" \n'.format(response.get('title'))
```

Comment:
This could perhaps be done a bit more simply with something like

```
to_write = '''
body = {response.get('body')}
user = {pr.user.login}
'''
```
etc.

Reply:
Don't I have to make it into an f string? Or I can just put it in brackets?